CHEWBBACA/chewBBACA.py

```diff
@@ -452,88 +924,483 @@ def msg(name=None):
+    parser.add_argument('-t', nargs='?', type=str, help='private token', required=False, default=False)
+    parser.add_argument('-m', nargs='?', type=str, help='tsv with metadata', required=False, default=False)
+    parser.add_argument('--mdr', nargs='?', type=str, help='maximum missing data allowed to fail, default 0.5 (50 percent missing data allowed). 1 == all profiles are uploaded even with 100 percent missing data', required=False, default=0.5)
+    parser.add_argument('--cpu', nargs='?', type=int, help='number of cpu', required=False, default=1)
+
+    args = parser.parse_args()
+
+    profileFile = args.p
+    pathSchema = args.s
+    token= args.t
+    metadata = args.m
+    cpu2use = args.cpu
+    percentMDallowed=args.mdr
+
+    send2NS.main(profileFile,pathSchema,token,metadata,percentMDallowed,cpu2use)
+
+
+def send_meta():
+
+    def msg(name=None):
+        return ''' chewBBACA.py SendMetadata [SendMetadata ...][-h] -s [S] -t [T] -p [P]
+                    '''
+
+    parser = argparse.ArgumentParser(description="send metadata to isolates on the NS",usage=msg())
+    parser.add_argument('SendMetadata', nargs='+', help='send metadata to isolates on the NS')
+    parser.add_argument('-t', nargs='?', type=str, help='private token', required=False, default=False)
+    parser.add_argument('-m', nargs='?', type=str, help='tsv with metadata', required=False, default=False)
+    parser.add_argument('--cpu', nargs='?', type=int, help='number of cpu', required=False, default=1)
+
+    args = parser.parse_args()
+
+    token= args.t
+    metadata = args.m
+    cpu2use = args.cpu
+
+    send_metadata.main(metadata,cpu2use,token)
+
+
+def down_prof():
+
+    def msg(name=None):
+        return ''' chewBBACA.py DownloadProfiles [DownloadProfiles ...][-h] --sp [SP] --sc [SC] --cpu [CPU]
+                    '''
+
+    parser = argparse.ArgumentParser(
+        description="Download profiles from the NS",usage=msg())
+    parser.add_argument('DownloadProfiles', nargs='+', help='download profiles from NS')
+    parser.add_argument('--sp', nargs='?', type=str, help='species uri', required=True)
+    parser.add_argument('--sc', nargs='?', type=str, help='schema id', required=True)
+    parser.add_argument('--cpu', nargs='?', type=int, help='number of cpu', required=False, default=1)
+    parser.add_argument('-r', nargs='?', type=str, help='genomes to down profile', required=False, default=None)
+    parser.add_argument('-p', nargs='?', type=str, help='profile with already downloaded profiles for that schema', required=False, default=None)
+    parser.add_argument('-t', nargs='?', type=str, help='private token', required=False, default=False)
+
+    args = parser.parse_args()
+
+    species = args.sp
+    schema = args.sc
+    cpu2use = args.cpu
+    genomes2Down = args.r
+    inputProfile = args.p
+    token = args.t
+
+    down_profiles.main(species,schema,cpu2use,inputProfile,genomes2Down,token)
+
```

Comment:
The function `down_prof` above should be commented or removed because it is calling a function from a deleted file. In the future, it can be added again to `chewBBACA.py`.